app/models/work.rb

```diff
@@ -31,6 +31,18 @@ def my_works(user)
       Work.where(created_by_user_id: user)
     end
 
+    def unfinished_works(user)
```

Comment:
@hectorcorrea I might have made this a bit differently.  You can chain methods, so you could have
```
def by_user(user)
  where(created_by_user_id: user)
end

def by_state(state)
  where(state: state)
end
```
The call could then be
```
@my_unfinished_works = Work.by_user(current_user).by_state("AWAITING-APPROVAL")
```
or are you doing this to hide the states in the work class?

The only reason I am pointing this out is creating the separate methods allow seams for filtering in the future.  Aka an administrator could filter by user, or by state.